home.css

```diff
@@ -0,0 +1,811 @@
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+}
+
+.artist-affiliation {
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+  font-size: 15px;
+  margin-top: 15px;
+  margin-bottom: 0;
+}
+
+.division-greyline {
+  color: gray;
+  font-size: 15px;
+  font-weight: 800;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.artist-bio {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  margin-top: 10px;
+  margin-bottom: 15px;
+}
+
+.open-menu {
+  height: 100vh;
+  width: 100%;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding-top: 100px;
+  background-color: rgba(49, 49, 49, 1);
+}
+
+.open-menu a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.top-nav {
+  width: 100%;
+  margin-bottom: 80px;
+}
+
+.top-nav #close-button {
+  display: block;
+  position: relative;
+  top: -70%;
+  left: 90%;
+  color: #ec5242;
+}
+
+.top-nav #close-button:hover {
+  cursor: pointer;
+}
+
+.top-nav ul {
+  width: 100%;
+  display: -webkit-box;
+  display: flex;
+  justify-content: space-evenly;
+  padding-left: 0;
+}
+
+.top-nav ul li {
+  list-style: none;
+  margin-bottom: 20px;
+}
+
+.top-nav ul li a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.bottom-nav {
+  width: 80%;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  gap: 30px;
+}
+
+.bottom-nav #nav-logo {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  margin-bottom: 50px;
+}
+
+.bottom-nav #nav-logo a {
+  color: #ec5242;
+  text-decoration: none;
+}
+
+.bottom-nav ul {
+  width: 100%;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  padding-left: 0;
+}
+
+#mellisa {
+  text-align: center;
+}
+
+#mellisam {
+  text-align: center;
+}
+
+.bottom-nav ul li {
+  list-style: none;
+}
+
+.bottom-nav #campaign {
+  display: none;
+  font-size: 20px;
+  border: 2px solid gray;
+  padding: 15px 25px;
+  margin-top: 10px;
+  color: #fff;
+}
+
+#division-line-div {
+  display: flex;
+  justify-content: center;
+  padding: 0;
+  margin: 0;
+}
+
+@media (min-width: 768px) {
+  nav {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    width: 90%;
```

Comment:
- [ ] Please make your header take the whole page width not boxed like you did (see image below) to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/169349324-ec887063-319d-4181-a3ac-0994315c918a.png)  | ![image](https://user-images.githubusercontent.com/68796232/169349359-5dbef86d-b875-402d-8bf7-dacc4e9e6849.png)  |

**[Hint]: You can do it just by changing the value of this width to be 100%**

Reply:
Noted @Omar-Muhamad, I had overlooked it.